website/docs/04 - Model/06 - Querying/02 - Query Operators.md

```diff
@@ -154,6 +154,10 @@ The query which will be executed on the MultiValue database is:
 select ITEM with DESCRIPTION like "...Bed..."
 ```
 
+:::caution
+Queries with the `$contains` operator cannot include single or double quotes (`'` or `"`) in the query constant value.
```

Comment:
Agreed.  I had thought about that for the work I started for improving deployment.  Probably the best strategy is to do a PR for the change and a separate PR immediately afterward for the docs.  The change would get merged in right away and the docs would remain a pending PR until its time to produce a release, at which time it'll be merged and then the release cut.
In this case, since a new release will need to happen right away the time difference won't matter much.